src/components/Tables/Artists.tsx

```diff
@@ -1,62 +1,139 @@
+                        void refetch()
+                    },
+                    onError: (e) => {
+                        setError(
+                            'Toggle featured failed. There was an error altering the database.'
+                        )
+                        console.error(e)
+                    }
+                }
+            )
+        },
+        [setFeaturedMutation, refetch]
+    )
+
+    const handleDelete = useCallback(
+        (artist: Artist) => {
+            deleteMutation(
+                { id: artist.id },
+                {
+                    onSuccess: () => {
+                        setError(null)
+                        void refetch()
+                    },
+                    onError: (e) => {
+                        setError(
+                            'Delete failed. There was an error altering the database.'
+                        )
+                        console.error(e)
+                    }
+                }
+            )
+        },
+        [deleteMutation, refetch]
+    )
+
+    const columns = useMemo(
         () => [
-            {
-                accessorKey: 'name',
+            columnHelper.accessor((row) => row.name, {
                 cell: (info) => info.getValue(),
-                header: () => <span>Name</span>
-            },
-            {
-                accessorKey: 'genre',
+                header: 'Name'
+            }),
+            columnHelper.accessor((row) => row.genre, {
                 cell: (info) => info.getValue(),
-                header: () => <span>Genre</span>
-            },
-            {
-                accessorKey: 'website',
+                header: 'Genre'
+            }),
+            columnHelper.accessor((row) => row.website, {
                 cell: (info) => info.getValue(),
-                header: () => <span>Website</span>
-            },
-            {
-                accessorKey: 'instagramHandle',
+                header: 'Website'
+            }),
+            columnHelper.accessor((row) => row.instagramHandle, {
                 cell: (info) => info.getValue(),
-                header: () => <span>Instagram</span>
-            },
-            {
-                accessorKey: 'active',
+                header: 'Instagram'
+            }),
+            columnHelper.accessor((row) => row.active, {
                 cell: (info) => info.renderValue()?.toString(),
-                header: () => <span>Active</span>,
+                header: 'Active',
                 enableColumnFilter: false
-            },
-            {
-                accessorKey: 'featured',
+            }),
+            columnHelper.accessor((row) => row.featured, {
                 cell: (info) => info.renderValue()?.toString(),
-                header: () => <span>Featured</span>,
+                header: 'Featured',
                 enableColumnFilter: false
-            }
+            }),
+            columnHelper.display({
```

Comment:
Pleasure ensure that the TableActionMenu has `cursor: pointer` on hover

Reply:
please explain.

Reply:
[css cursor rule](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor )

When a user hovers their cursor over the TableActionMenu it should change to a `pointer` type indicating that the element is clickable - just like an anchor